basket/news/management/commands/process_donations_queue.py

```diff
@@ -15,13 +15,15 @@
 from basket.news.tasks import (
     process_donation,
     process_donation_event,
+    process_newsletter_subscribe,
     process_petition_signature,
 )
 
 
 EVENT_TYPES = {
     'donation': process_donation,
     'crm_petition_data': process_petition_signature,
+    'newsletter_signup_data': process_newsletter_subscribe,
```

Comment:
Out of curiosity, where are these used?

Reply:
What are the `these` in this question? These `EVENT_TYPES`? If so they're used just a few lines below this in this file.

Reply:
Ah, I see. I did a search for the types and only found `donation`. The other types are specified in the incoming data. Got it.